test/PriceFeed.gas.test.ts

```diff
@@ -0,0 +1,113 @@
+import { parseEther } from "ethers/lib/utils"
+import { ethers, waffle } from "hardhat"
+import { BandPriceFeed, ChainlinkPriceFeed, TestAggregatorV3, TestPriceFeed, TestStdReference } from "../typechain"
+
+interface PriceFeedFixture {
+    bandPriceFeed: BandPriceFeed
+    bandReference: TestStdReference
+    baseAsset: string
+
+    // chainlinik
+    chainlinkPriceFeed: ChainlinkPriceFeed
+    aggregator: TestAggregatorV3
+}
+
+async function priceFeedFixture(): Promise<PriceFeedFixture> {
+    // band protocol
+    const testStdReferenceFactory = await ethers.getContractFactory("TestStdReference")
+    const testStdReference = await testStdReferenceFactory.deploy()
+
+    const baseAsset = "ETH"
+    const bandPriceFeedFactory = await ethers.getContractFactory("BandPriceFeed")
+    const bandPriceFeed = (await bandPriceFeedFactory.deploy(testStdReference.address, baseAsset)) as BandPriceFeed
+
+    // chainlink
+    const testAggregatorFactory = await ethers.getContractFactory("TestAggregatorV3")
+    const testAggregator = await testAggregatorFactory.deploy()
+
+    const chainlinkPriceFeedFactory = await ethers.getContractFactory("ChainlinkPriceFeed")
+    const chainlinkPriceFeed = (await chainlinkPriceFeedFactory.deploy(testAggregator.address)) as ChainlinkPriceFeed
+
+    return { bandPriceFeed, bandReference: testStdReference, baseAsset, chainlinkPriceFeed, aggregator: testAggregator }
+}
+
+describe.skip("Price feed gas test", () => {
```

Comment:
maybe forgot to remove skip?

Reply:
I skip it on purpose. This test should be run by CI, it just for our gas estimation.